Classes/Queue/RabbitQueue.php

```diff
@@ -117,6 +117,21 @@ public function __construct(string $name, array $options = [])
         $queueOptions = $options['queueOptions'];
 
         $this->queueName = $queueOptions['name'] ?? $this->name;
+
+        $this->useDLX = (bool) ($options['useDLX'] ?? false);
+        if ($this->useDLX
+            && is_string($options['useDLX'])
+            && strtolower($options['useDLX']) !== 'true'
+        ) {
+            $queueOptions['arguments'] = $queueOptions['arguments'] ?? [];
+            if (! isset($queueOptions['arguments']['x-dead-letter-exchange'])) {
+                $queueOptions['arguments']['x-dead-letter-exchange'] = $options['useDLX'];
+            }
+            if (! isset($queueOptions['arguments']['x-dead-letter-routing-key'])) {
+                $queueOptions['arguments']['x-dead-letter-routing-key'] = 'queue.' . $this->queueName;
```

Comment:
The `queue.` must be set fixed? Shouldn't this be flexible? https://www.rabbitmq.com/dlx.html#using-optional-queue-arguments does not say something about a fixed queue prefix

Reply:
I don't think the `queue.` part is needed at all. Actually, when removing it from here, the explicit binding in line 152 might actually be unnecessary.

When using `amq.direct` (aka the default exchange) as `x-dead-letter-exchange` and only the queue name as `x-dead-letter-routing-key`, the message should be put back in the original queue without manually adding a binding.

[From RabbitMQ docs](https://www.rabbitmq.com/tutorials/amqp-concepts.html#exchange-default): 

> The default exchange is a direct exchange with no name (empty string) pre-declared by the broker. It has one special property that makes it very useful for simple applications: every queue that is created is automatically bound to it with a routing key which is the same as the queue name.

This means that a binding is only needed if the routing key is specified by the user

Reply:
Sorry for the fuzz. The point of this PR was actually a misconfiguration on my end.

What i was actually trying to do was "retry after x seconds". So I had a topic exchange "retry-delay.dlx" which had DLX set to "amq.direct" and a certain TTL.

I was under the impression the "queue." prefix was rabbit best practice, which is actually not the case.

I didn't realize there's a difference between an exchange called "amq.direct" and "".
Both are provided by rabbit as default direct exchanges but only the one with no name has the special feature of bouncing messages back to a queue named exactly like the routing key.

For the default direct exchange "amq.direct" this is not the case, so consuming queues need to be bound manually. 

That's why I added the manual bind. When using not "amq.direct" but "" as DLX for my "retry-delay.dlx" no manual binding is required.

The whole thing can be solved just now with no additional PHP code with config like this:
```
Flowpack:
  JobQueue:
    Common:
      queues:
        'queue-name':
          useDLX: true
          routingKey: 'routing-key'
          queueOptions:
            name: 'routing-key'
            arguments:
              x-dead-letter-exchange: retry-delay.dlx
```
(Assuming the topic exchange "retry-delay.dlx" I used here doesn't DLX to "amq.direct" but to "", ofc.)

I'm closing this PR because it's useless :).